code/obj/table.dm

```diff
@@ -265,7 +273,10 @@
 				return
 
 		else if (iswrenchingtool(W) && !src.status) // shouldn't have status unless it's reinforced, maybe? hopefully?
-			actions.start(new /datum/action/bar/icon/table_tool_interact(src, W, TABLE_DISASSEMBLE), user)
+			if (!istype(src, /obj/table/folding))
```

Comment:
probably flip this around - more clear on behavior

Reply:
Got it. Fixed.